static/app/views/issueDetails/streamline/solutionsHubDrawer.tsx

```diff
@@ -103,9 +103,8 @@ function AutofixStartBox({onSend, groupId}: AutofixStartBoxProps) {
 
 const shouldDisplayAiAutofixForOrganization = (organization: Organization) => {
   return (
-    ((organization.features.includes('autofix') &&
-      organization.features.includes('issue-details-autofix-ui')) ||
-      organization.genAIConsent) &&
+    organization.features.includes('gen-ai-features') &&
+    organization.genAIConsent &&
```

Comment:
No, this was (and probably still technically is) needed for GitHub Copilot extension users. (yes it is circumventing our flag, yes it is bad)

Reply:
in the UI?

Reply:
this would make autofix show up for users that aren't in the rollout or flagged, which is undesired behavior if we want to ban anyone in the future too (yes we have banned orgs from flags, especially ones that abused)

Reply:
Offline decision: it's ok to remove this workaround for Copilot